### ISSUE TITLE
Events

### ISSUE THREAD

**Issue by front-stream** [zyramedia/ionic-image-loader#24]
Thanks for cool plugin.

Is it possible to add custom class or detect when image is downloaded? I would like to display image using 'fade-in' effect.

**Comment by ihadeed** [comment]
Sure I can add this.

**Comment by front-stream** [comment]
Thanks, I am waiting for this feature

**Comment by ihadeed** [comment]
Just published `1.7.0` that has this feature.

To do what you're looking to do:
```html
<img-loader src="path/to/img" (load)="onImageLoad($event)"></img-loader>
```
```ts
import {Renderer } from '@angular/core';
import { ImgLoader } from 'ionic-image-loader';

...

constructor(private renderer: Renderer) { }

...

onImageLoad(imgLoader: ImgLoader) {
  this.renderer.setElementClass(imgLoader.element, 'fade-in', true);  
}
```

**Comment by front-stream** [comment]
Thank you! works like a charm. Cheers
